services/notification/notifier.go

```diff
@@ -1,7 +1,7 @@
 // Copyright 2018 The Gitea Authors. All rights reserved.
 // SPDX-License-Identifier: MIT
 
-package ui
+package notification
```

Comment:
What's the difference between `services/notification`, and `services/notify`?
That `…ication` handles the UI code for notifications, while `notify` enables the general notification methods?
We should probably clarify this somewhere.

Reply:
Or rename this package to `uinotification`?